.github/workflows/main.yml

```diff
@@ -284,7 +284,7 @@ jobs:
         run: |
           ## SETUP ENVIRONMENT SCRIPT
           $CommitHash = (git rev-parse HEAD)[0..8] -join ''
-          Write-Output "::set-output name=commitHash::${CommitHash}"
+          Write-Output "commitHash=${CommitHash}" >> $GITHUB_OUTPUT
```

Comment:
$env.GITHUB_OUTPUT when dealing with pwsh?

Reply:
This was discussed on a separate but related PR on a different repository, but yes, the syntax according to the documentation should be:
```
"commitHash=${CommitHash}" >> $env:GITHUB_OUTPUT
```